agent.go

```diff
@@ -870,9 +870,19 @@ func (p *pod) runContainerProcess(cid, pid string, terminal bool, started chan e
 		fieldLogger.Info("Got process state")
 
 		if waitStatus, ok := processState.Sys().(syscall.WaitStatus); ok {
-			exitCode = uint8(waitStatus.ExitStatus())
-			fieldLogger.WithField("exit-code", exitCode).Info("got wait exit code")
+			exitStatus := waitStatus.ExitStatus()
+
+			if waitStatus.Signaled() {
+				exitCode = uint8(128 + waitStatus.Signal())
```

Comment:
could you please define a variable and document why 128 ?

Reply:
@devimc changed to constant and documented :D